openedx/adg/lms/applications/test/test_forms.py

```diff
@@ -0,0 +1,93 @@
+"""
+All tests for applications views
+"""
+from datetime import date, timedelta
+from unittest.mock import Mock
+
+import pytest
+from dateutil.relativedelta import relativedelta
+
+from openedx.adg.lms.applications.forms import ContactInformationForm
+from openedx.adg.lms.applications.test.factories import ExtendedUserProfileFactory
+from student.tests.factories import UserFactory
+
+
+def contact_information_dictionary(name='test', email='test@example.com', city='XYZ',
+                                   gender='male', phone_number='00000000',
+                                   birth_day=None, birth_month=None, birth_year=None,
+                                   organization='test', linkedin_url=''):
+    """
+    Initialize the data dictionary for contact information forms
+    """
+    return {
+        'name': name,
+        'email': email,
+        'city': city,
+        'saudi_national': True,
+        'gender': gender,
+        'phone_number': phone_number,
+        'birth_day': birth_day,
+        'birth_month': birth_month,
+        'birth_year': birth_year,
+        'organization': organization,
+        'linkedin_url': linkedin_url
+    }
+
+
+@pytest.mark.django_db
+def create_factories():
+    """
+    Create user and related factories
+    """
+    user = UserFactory.create()
+
+    extended_profile = ExtendedUserProfileFactory()
+    extended_profile.user = user
+    extended_profile.save()
```

Comment:
No need for this. When we will create extended profile using factory it will create a user as we have provided the sub factory in extended profile factory